.github/CODEOWNERS

```diff
@@ -778,6 +778,7 @@ packages/kbn-yarn-lock-validator @elastic/kibana-operations
 
 # Observability robots
 /.github/workflows/deploy-my-kibana.yml @elastic/observablt-robots
+/.github/workflows/oblt-github-commands @elastic/observablt-robots
```

Comment:
@elastic/kibana-operations , for some reason i see this error, but this team is available at the Elastic org level, do you happen to know how to enable it at the repo level?

Reply:
Done!